odoo/addons/base/rng/graph_view.rng

```diff
@@ -23,6 +23,8 @@
             <rng:optional><rng:attribute name="order"/></rng:optional>
             <rng:optional><rng:attribute name="disable_linking"/></rng:optional>
             <rng:optional><rng:attribute name="sample"/></rng:optional>
+            <rng:optional><rng:attribute name="measure"/></rng:optional>
```

Comment:
seems very weird to add those attributes.
From what I can gather, the initial groupBys of the graph are retrieved either from the visible fields in the view or from the context
Same fore measures

I'm not in favor of adding yet another way to define groupBys on the graph view, not in the pivot view

To confirm with  @Polymorphe57

Reply:
I think @kebeclibre is right. The measure is determined in the arch by the last field with attribute type="measure" and the groupby by the fields without attribute type="measure" that are visible.
Example:

`
  <graph>
    <field name="name_1" type="measure"/>
    <field name="name_2" type="measure"/>
    <field name="name_3"/>
    <field name="name_4" invisible="1"/>
    <field name="name_5"/>
  </graph>
`
 gives measure= name_2 and group by = [name_3, name_5].
So it is unnecessary to add new root node attributes for that.
I think the advantages of defining parameters via field tags is that some validation is done when saving the arch.